backend/src/main/java/ai/verta/modeldb/utils/RdbmsUtils.java

```diff
@@ -797,7 +797,8 @@ public static Predicate getValuePredicate(
         LOGGER.debug("Called switch case : string_value");
         if (!value.getStringValue().isEmpty()) {
           LOGGER.debug("Called switch case : string value exist");
-          if (fieldName.equals(ModelDBConstants.ATTRIBUTES)) {
+          if (fieldName.equals(ModelDBConstants.ATTRIBUTES)
```

Comment:
can you elaborate on what was the bug and how fixing ATTRIBUTES if fixes the bug?